lms/djangoapps/teams/static/teams/js/views/rocket_chat_discussion.js

```diff
@@ -0,0 +1,15 @@
+/* Javascript for RocketChat- discussions */
+(function(define) {
+    'use strict';
+    define(['jquery'],
+
+        function($) {
+
+            var xblock = $(".xblock-student_view-rocketc");
+            var style = $("style");
+            $( "body" ).empty();
+            $( "body" ).append(xblock);
+            $( "body" ).append(style);
+
+        });
+}).call(this, define || RequireJS.define);
```

Comment:
nit: missing new line at the end of the file

Reply:
done